debezium-server/debezium-server-nats_streaming/src/main/java/io/debezium/server/nats_streaming/NatsStreamingChangeConsumer.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.nats_streaming;
+
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.DebeziumException;
+import io.debezium.engine.ChangeEvent;
+import io.debezium.engine.DebeziumEngine;
+import io.debezium.engine.DebeziumEngine.RecordCommitter;
+import io.debezium.server.BaseChangeConsumer;
+import io.debezium.server.CustomConsumerBuilder;
+import io.nats.client.Connection;
+import io.nats.client.ConnectionListener;
+import io.nats.streaming.NatsStreaming;
+import io.nats.streaming.Options;
+import io.nats.streaming.StreamingConnection;
+
+/**
+ * Implementation of the consumer that delivers the messages into NATS Streaming subject.
+ *
+ * @author Thiago Avancini
+ */
+@Named("nats_streaming")
+@Dependent
+public class NatsStreamingChangeConsumer extends BaseChangeConsumer
+        implements DebeziumEngine.ChangeConsumer<ChangeEvent<Object, Object>> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NatsStreamingChangeConsumer.class);
+
+    private static final String PROP_PREFIX = "debezium.sink.nats_streaming.";
+    private static final String PROP_URL = PROP_PREFIX + "url";
+    private static final String PROP_CLUSTER_ID = PROP_PREFIX + "cluster.id";
+    private static final String PROP_CLIENT_ID = PROP_PREFIX + "client.id";
+
+    private String url;
+    private String clusterID;
```

Comment:
PLease use calmel case so `clusterId` same for the next var.

Reply:
Done!

Reply:
Done!